Index.html

```diff
@@ -4,281 +4,379 @@
+          </div>
+          <div class="Frameworks">
+              <div class="language">
+                  <img alt="Vector" class="orange-vector" src="images/img_5.png">
+                  <h2 class="lang-h2">Frameworks</h2>
+              </div>
+              <div class="language-names">
+                  <ul class="language-list">
+                      <li class="language-list-content">
+                          Bootstrap
+                      </li>
+                      <li class="language-list-content">
+                          Ruby on Rails
+                      </li>
+                      <li class="language-list-content">
+                          RSPec
+                      </li>
+                      <li class="language-list-content">
+                          CapyBara
+                      </li>
+                      <li class="language-list-content">
+                          Selenium
+                      </li>
+                  </ul>
+              </div>
+          </div>
+          <div class="skill">
+              <div class="language skills">
+                  <img alt="Vector" class="orange-vector" src="images/img_6.png">
+                  <h2 class="lang-h2">Skills</h2>
+              </div>
+              <div class="language-names">
+                  <ul class="language-list">
+                      <li class="language-list-content">
+                          Codekit
+                      </li>
+                      <li class="language-list-content">
+                          Github
+                      </li>
+                      <li class="language-list-content">
+                          Codepen
+                      </li>
+                      <li class="language-list-content">
+                          Gitlab
+                      </li>
+                      <li class="language-list-content">
+                          Terminal
+                      </li>
+                  </ul>
+              </div>
+          </div>
+      </div>
+  </section>
+  <section class="lastpage">
+      <div class="contact-form">
+          <h2 class="lastpage-heading">I'm always interested in hearing about new projects, so if you'd like to chat
+              please get in touch.</h2>
+          <form class="form" action="https://formspree.io/f/xnqwwrzy" method="POST">
+            <div class="same-row">
+              <input class="fullname same-line" type="text" name="fullname" placeholder="Full name" required>
+              <input class="fullname same-line" type="text" name="lastname" placeholder="Last Name" required>
+            </div>
+              
+              <input class="fullname" type="email" name="fullname" placeholder="Email address" required>
+              <textarea class="text-area" placeholder="Enter text here" required></textarea>
+              <button type="submit" class="animate-btn button1 button1-text button-end btn-d">Get in touch</button>
+          </form>
```

Comment:
- [ ] Your contact section is not identical to the figma design as per requirements and suggested changes by the previous reviewer. 
  - kindly consider to a lot 40% width to the left heading in this section and 60% width to the left side form sub- 
   section.
  - Also make sure to give appropriate margins to these sub-sections as in the figma design 
  - You can also choose to take one input for the Full name (just like the mobile version) as it is flexible by microverse.
  figma design
![t](https://github.com/MohammadYaser/Portfolio/assets/96683843/84e89514-3d86-487c-92af-c3083b099479)